class InstantiateCSVError(Exception):
    """Класс исключения при повреждении файла"""
    pass
